src/lib/Components/Bidding/Screens/__tests__/BidResult-tests.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react"
+import "react-native"
+
+import * as renderer from "react-test-renderer"
+import { BidResult } from "../BidResult"
+
+it("renders winning screen properly", () => {
+  jest.useFakeTimers()
+
+  const bg = renderer.create(<BidResult winning />).toJSON()
+
+  expect(bg).toMatchSnapshot()
+  expect(setInterval).toHaveBeenCalledTimes(1)
```

Comment:
@l2succes This is the test I needed to use `jest.useFakeTimers()` and `expect(setInterval)` otherwise the test just hangs. I'm not sure if I'm doing this right. Let me know if there's a better way to test this!